docs/getting-started/sensor-full-disk-access.asciidoc

```diff
@@ -1,25 +1,93 @@
+Elastic Endpoint Security requires Full Disk Access to protect you from malware and other cybersecurity threats. Full Disk Access permissions is a new privacy feature introduced in macOS Mojave (10.14) that prevents some applications from accessing your data. This means you need to manually grant permission for the Endgame sensor and Elastic Endpoint to access these protected areas of your Mac.
 
-This article describes how to enable full disk access for both the Elastic Agent, required in order to enable Elastic Endpoint Security, and the legacy Endgame sensor.
+This article describes how to enable Full Disk Access for the required security system extensions, the Elastic Endpoint sensor, and the legacy Endgame sensor.
+
+[discrete]
+[[macos-permissions]]
+== macOS permissions
+
+The behavior of the Endgame sensor and Elastic Endpoint differs based on your macOS version. MDM/JAMF users can pre-approve all Full Disk Access without granting permission to the sensors. However, depending on the macOS version and sensor type, non-MDM/JAMF users may be prompted to enable Full Disk Access for required security files.
+
+**Endgame Sensor**
+
+- `10.13, 10.14, 10.15`: Users cannot proceed with installation without first granting the sensor the ability to load a <<enable-kernel-extension,kernel extension>>. During installation, you will be prompted to go to **System preferences** and approve loading the kernel. Upon approval, installation proceeds. 
+
+- `11.0` (Big Sur): Users cannot proceed with installation without first granting the sensor the ability to load <<system-extension>>. During installation, you will be prompted to go to **System preferences** and approve loading the system extension. Upon approval, a second prompt appears to enable **Network Filtering**. Approve this final prompt for installation to proceed.
++
+You also must grant Full Disk Access to `com.endgame.systemextension`.
+
+- `10.14.6+, 10.15, 11.0`: Grant the <<endpoint-endgame-sensor,esensor>> Full Disk Access.
+
+
+**Elastic Endpoint**
+
+No prompts appear to approve the <<enable-kernel-extension,kernel>>, <<system-extension>>, or the <<endpoint-endgame-sensor,elastic-endgame>>, due to installation happening through the {agent}. After installation, Endpoint policies **will fail** to detect events until you approve and enable kernel or system extension loading and Full Disk Access for each version, as reflected in the **Administration > Endpoints** page of the security application.
+
+- `10.13, 10.14, 10.15`: Approve the <<enable-kernel-extension,kernel>>.
+
+- `11.0` (Big Sur): Give Full Disk Access to the <<system-extension>>, as well as `co.elastic.systemextension`.
+
+- - `10.14.6+, 10.15, 11.0`: Grant the <<endpoint-endgame-sensor,elastic-endgame>> Full Disk Access.
```

Comment:
```suggestion
- `10.14.6+, 10.15, 11.0`: Grant the <<endpoint-endgame-sensor,elastic-endgame>> Full Disk Access.
```
Why are we calling this elastic-endgame? They are either giving FDA to the `elastic-endpoint` or `esensor` file right?